job_runner/reporting/file.py

```diff
@@ -76,28 +95,37 @@ def report(self, msg):
                 self._dump_error_file(msg.job, error_msg)
 
         elif isinstance(msg, Running):
-            job_status["max_memory_usage"] = msg.max_memory_usage
-            job_status["current_memory_usage"] = msg.current_memory_usage
-            job_status["status"] = _JOB_STATUS_RUNNING
+            job_status_insert(
+                max_memory_usage=msg.max_memory_usage,
+                current_memory_usage=msg.current_memory_usage,
+                status=_JOB_STATUS_RUNNING,
+            )
```

Comment:
```suggestion
            job_status.extend({
                "max_memory_usage": msg.max_memory_usage,
                "current_memory_usage": msg.current_memory_usage,
                "status": _JOB_STATUS_RUNNING,
            })
```
Maybe? Then you don't really need the inline function.

Reply:
Inline function removed